application/templates/mail/request_from_committed_to_accepted.plush.html

```diff
@@ -1,6 +1,12 @@
-<p><%= receiverNickname %> has accepted your offer to fulfill a request on <a href="<%= uiURL %>"><%= appName %></a>.</p>
 <h4><a href="<%= postURL %>"><%= postTitle %></a></h4>
 
-<p><%= postDescription %></p>
-<br>
-<br>
+<p>
+    <%= receiverNickname %> has accepted your offer to fulfill their request. Be sure to communicate with them
+    to coordinate the details. For guidance on things to coordinate, see the
+    <a href="">Responsibilities of Providers</a> section of our Terms of Use.
+</p>
+
+<p>
```

Comment:
Is it worth using a partial for this (and for the provider separately), since they get repeated so much?

`<%= partial("request_communicate_receiver.html") %>`